core/bones/base.py

```diff
@@ -339,13 +339,16 @@ def fromClient(self, skel: 'SkeletonInstance', name: str, data: dict) -> Union[N
             for language in self.languages:
                 res[language] = []
                 if language in parsedData:
-                    for singleValue in parsedData[language]:
+                    for idx,singleValue in enumerate(parsedData[language]):
                         if self.isEmpty(singleValue):
                             continue
                         isEmpty = False
                         parsedVal, parseErrors = self.singleValueFromClient(singleValue, skel, name, data)
                         res[language].append(parsedVal)
                         if parseErrors:
+                            for parseError in parseErrors:
+                                parseError.fieldPath.insert(0, language)
+                                parseError.fieldPath.insert(1, str(idx))
```

Comment:
I convert the index to str because in the ReadFromClientError we set fieldPath to a list of str.
https://github.com/viur-framework/viur-core/blob/33f15adda4dcb0296fa3ee58faeed50b4c9f3696/core/bones/base.py#L34-L39